GitUI/ScriptsEngine/ScriptsManager.ScriptRunner.cs

```diff
@@ -27,6 +32,70 @@ public static bool RunScript(ScriptInfo script, IWin32Window owner, IGitUIComman
                 }
             }
 
+            private static (string? arguments, bool abort, bool cancel) ParseUserInputs(string scriptName, string? arguments, IGitUICommands uiCommands, IWin32Window owner, IScriptOptionsProvider? scriptOptionsProvider = null)
```

Comment:
Do we have tests for this?

Reply:
Yes, but they fail at the moment because they need to call `ParseUserInputs` instead of `ScriptOptionsParser.Parse`.